UserBundle/Document/Group.php

```diff
@@ -7,7 +7,7 @@
 
 /**
  * @ODM\Document(
- *  collection="group_document",
+ *  collection="group",
```

Comment:
Ok I understand now why this strange naming compared to the others documents.
I'm not a fan of group_document as it's the only collection to have such naming, so can I propose an alternative such as users_group or rights_group ? It avoids the basic naming conflict plus the unique and artificial suffix "document", but upgrades the meaning.